beheer.md

```diff
@@ -12,29 +12,58 @@ In dit document worden allerlei procedures en overzichten beschreven die van bel
 - [Openen Slack kanaal](#openen-slack-kanaal)
 - [Openen OpenAPI standaard pagina op GEMMA Online](#openen-openapi-standaard-pagina-op-gemma-online)
 - [Openen GEMMA Online discussieforum](#openen-gemma-online-discussieforum)
 - [Werken met Docker](#werken-met-docker)
 
+
+
+## Overdragen van API-development naar API-beheer
+* Er moet een template komen van hoe een API standaard wordt overgedragen! Nu ontbraken bijvoorbeeld linkjes naar documentatie.
+* Criteria (checklist) moet ook naar API-developers
+
+## Starten van API ontwerp en ontwikkeling
+Beheer helpt API developers/designers door:
+* Beheer geeft toegang tot Dockerhub voor pushen van images aan devs
+* Beheer vraagt aan devs: Voordat je oplevert, hang in test platform met deze credentials
+* Documenteer altijd bepaalde onderwerpen; als het n.v.t. is, moet dat er ook staan (bijv. afhankelijkheden van andere APIs)
+* Geef via email naar "..." aan dat de standaard in beheer genomen moet worden.
+
 ## Criteria voor in beheername
 Voordat API standaarden in beheer genomen kunnen worden moeten deze voldoen aan een aantal eisen. Hieronder staat een eerste aanzet daartoe:
 
-* De standaard moet formeel goedgekeurd zijn.
-* API specificatie (OAS3) moet beschikbaar zijn en uit de referentie implementatie of uit de imvertor genereerbaar zijn (afhankelijk van de keuze van het project).
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
+* Er moet een testomgeving draaien van de implementatie.
+*  Is de standaard (deze bestaat uit de OAS, referentieimplementatie, documentatie) formeel goedgekeurd, concreet gemaakt:
+  * Er moet een API-lab gehouden zijn van deze API
+  * Er moeten één of meer tussetijdse (sprint) demo's zijn geweest van deze API
+  * De API moet beschikbaar zijn in test platform
+* De standaard moet publiekelijk beschikbaar zijn:
+  * De OAS moet zonder restricties op te vragen zijn in de browser
+* Kwaliteitscheck op OAS:
+  * De standaard moet voldoen aan de landelijke API strategie, of er is gedocumenteerd en beargumenteerd afgeweken.
+  * Openen in Swaggerhub ter dubbelcheck
+* Documentatie omvat minimaal:
+  * Een UML model van het informatiemodel (de objecttypen en onderlinge relaties) zoals deze in de API gebruikt wordt.
+  * Een overzicht van welke gegevens worden ontsloten in de vorm van een tabel van de resource + bijzonderheden. 
+  * INSTALL.rst van de ref. impl.
+  * Een beschrijving van de business logica die in de standaard van toepassing is.
+  * Een overzicht van wijzigingen (changelog) met daarin per versie wat is toegevoegd, gewijzigd en hoe te migreren van de voorgaande naar deze versie.
+  * De standaard bevat een overzicht waarin wordt aangegeven welke versies van de betreffende API standaard met welke versies van andere API standaarden compatible zijn.
+ * De standaard bevat een overzicht waarin alle partijen en personen die bij de ontwikkeling van de standaard betrokken zijn (geweest), zijn opgenomen.
```

Comment:
```suggestion
  * De standaard bevat een overzicht waarin alle partijen en personen die bij de ontwikkeling van de standaard betrokken zijn (geweest).
```

Reply:
Mijn zin kan beter maar in jouw voorstel ontbreekt het werkwoord?

Reply:
Inmiddels heb ik de zin aangepast, kun je kijken of dit beter is? Dan zou dat het laatste puntje zijn voor de merge.